src/OceanSeaIceModels/OceanSeaIceModels.jl

```diff
@@ -45,6 +45,14 @@ sea_ice_concentration(sea_ice::SeaIceSimulation) = sea_ice.model.ice_concentrati
 ##### Some implementation
 #####
 
+# Atmosphere interface
+interpolate_atmosphere_state!(interfaces, atmosphere, coupled_model) = nothing
+compute_net_atmosphere_fluxes!(copuled_model) = nothing
```

Comment:
I think this is already defined in the top file (`OceanSeaIceModels.jl`)

Reply:
IT is defined for PrescribedAtmospheres but not a fallback. I'd like to lay the groundwork for distinguishing between the interface implementation, and PRescribedAtmospheres.